IDE/ARDUINO/sketches/wolfssl_client/wolfssl_client.ino

```diff
@@ -40,6 +48,7 @@ void setup() {
   WOLFSSL_METHOD* method;
 
   Serial.begin(9600);
+	 delay(4000);
```

Comment:
please leave a comment here for the next person (or when you're coming back later) why delay(4000) is used